herd/tests/instructions/ASL-pseudo-arch/for-toofar.litmus

```diff
@@ -5,19 +5,19 @@ x = 0;
 0:X1=x;
 }
 
-func T0(x::bits(32))
+func T0(x::bits(64))
 begin
   var s: integer = 0;
-  let imax = UInt(read_memory(x, 32));
+  let imax = UInt(read_memory{32}(x));
   for i = 1 to imax do
     s = s + i;
   end;
 end;
 
-func T1(x::bits(32))
+func T1(x::bits(64))
 begin
   for k = 1 to 3 do
-    write_memory(x,32,k[2:0]);
+    write_memory{32}(x,k[2:0]);
```

Comment:
This one is an old one. Interestingly the fix is `write_memory{32}(x,k[31:0])` because the width of any AArch64ASL access is checked to be 32, 64, or 128...